### DISCUSSION TITLE
how to save hls to path

### DISCUSSION THREAD

**Discussion by seamanm** [bluenviron/mediamtx#1558] [Questions]
hi all:
i need config a path save hls's ts and m3u8,how to ?

**Comment by aler9** [comment]
Hello, there's a new option called `hlsDirectory` that allows to store segments in a folder:

```yml
hlsDirectory: /tmp/mediamtx
```

However the M3U8 playlist is not saved to disk but kept in RAM. This is because the playlist is updated multiple times per second (when Low Latency-HLS is enabled) and writing it to disk would result in too many writes. Nonetheless you can generate a new playlist by listing and parsing segments stored in the folder.

[a comment by an automated account is not shown]
